README.md

```diff
@@ -63,9 +66,9 @@
 
 > Describe between 1-3 key features of the application.
 
```

Comment:
- [optional] Please kindly remove all the placeholders in your README file or you customize them as appropriate. The placeholders are there to describe what to do with each header. 👍

Reply:
Thanks @OyePriscilla For reviewing this PR! I'll do it!

Reply:
@EddyShimwa Great! You're welcome.